runtime/autoload/health/nvim.vim

```diff
@@ -144,6 +144,23 @@ function! s:check_performance() abort
           \ ['Install a different Nvim package, or rebuild with `CMAKE_BUILD_TYPE=RelWithDebInfo`.',
           \  s:suggest_faq])
   endif
+
+  " check for slow shell invocation
+  let shell_cmd = 'echo'
+  let slow_cmd_time = 1.5
+  if !executable(shell_cmd)
```

Comment:
as @matveyt noted, this check is probably counterproductive.

Reply:
Removed